specifyweb/workbench/upload/tests/testuploading.py

```diff
@@ -173,15 +173,15 @@ def test_enforced_county(self) -> None:
             results[0].record_result,
             FailedBusinessRule(
                 message='missingRequiredTreeParent',
-                payload={'names':['County']},
+                payload={'names':"['County']"},
```

Comment:
This doesn't look right. Front-end expects this to be a list, not a json string

Reply:
ya, but the test fails with an assertion error without this change

Reply:
This should be fixed somehow before we can merge the PR, or else it will cause front-end to fail